.gitignore

```diff
@@ -33,6 +33,7 @@ models/*
 /benchmark-matmult
 /vdot
 /Pipfile
+/mulmat-tune
```

Comment:
```suggestion
```

Reply:
Add a `.gitignore` in `examples/mulmat-tune` instead:

```
analyze/
bench-out/
```

Delete all those files.